crates/server/src/admin/service.rs

```diff
@@ -111,11 +112,15 @@ pub(crate) fn setup(
                 .post(handlers::context::create_context_handler),
         )
         .route("/dev/applications", get(list_applications_handler))
-        .layer(Extension(shared_state));
+        .route_layer(axum::middleware::from_fn(
+            middleware::dev_auth::dev_mode_auth,
+        ));
 
     let admin_router = Router::new()
-        .nest("/", unprotected_router)
-        .nest("/", protected_router)
+        .merge(unprotected_router)
+        .merge(protected_router)
+        .merge(dev_router)
```

Comment:
Do you still need dev routes? Can you use regular routes?

Reply:
I need them because they don't have the same auth as the regular routes. Also, for example, `/dev/install-application` and `/install-application` don't install the app in the same way